modules/reitit-core/project.clj

```diff
@@ -0,0 +1,7 @@
+(load-file "../../.deps-versions.clj")
+(defproject metosin/reitit-core reitit-version
+  :description "Snappy data-driven router for Clojure(Script)"
+  :url "https://github.com/metosin/reitit/tree/master/modules/reitit-core"
```

Comment:
Maybe the URL should just point to reitit root? It will be shown in Clojars and elsewhere and I think that's where I'd like to go.

Reply:
fixed